iocore/cache/CacheRead.cc

```diff
@@ -1175,6 +1175,20 @@ CacheVC::openReadStartHead(int event, Event *e)
       } else {
         f.single_fragment = false;
       }
+
```

Comment:
I wonder if this should be checked when selecting alternates, but then it wouldn't get cleaned up as well.